cvss/parser.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+from cvss import CVSS3, CVSS2
+from cvss.exceptions import CVSSError
+
+
+def parse_cvss_from_text(text):
+    # Vector may start with 'CVSS:3.0' substring - which is not extracted.
+    # Vector starts and ends with capitals
+    # which makes parsing from incorrectly spaced words easier.
+    matches = re.compile(r'((?:CVSS:3\.0/)?[A-Z][A-Za-z:/]+[A-Z])').findall(text)
+
+    min_vector_length = 26
```

Comment:
I think it would be better to define a constant above the function and add description why 26 is optimal number.